pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.ruff]
```

Comment:
Can you add the other fields to pyproject.toml as well please? You can use uv init (might pull in this issue #99 )

Reply:
yes i can work on that issue after this one

Reply:
That would be awesome, thanks.